lib/pages/chat/events/message_content.dart

```diff
@@ -203,7 +203,7 @@ class MessageContent extends StatelessWidget {
                 padding: const EdgeInsets.only(left: 8.0, right: 8.0),
                 child: HtmlMessage(
                   html: html,
-                  defaultTextStyle: TextStyle(
+                  defaultTextStyle: Theme.of(context).textTheme.bodyLarge?.copyWith(
```

Comment:
this ticket for `Sender name`, but only see the change for the `content`. It is hard for reviewing. Please keep your logic focus to the scope of ticket